cmd/syft/cli/options/format_syft_json.go

```diff
@@ -0,0 +1,33 @@
+package options
+
+import (
+	"github.com/anchore/syft/syft/format/syftjson"
+	"github.com/anchore/syft/syft/sbom"
+)
+
+type FormatSyftJSON struct {
```

Comment:
The upside with two structs is that we have the CLI-facing API that has it's own facade -- the struct tags defined here affect how information is represented in a yaml configuration and mapped for configuration processing. There is a separate struct that is within the core library that can have more values and does not have any struct tags that affect application configuration. This is a strong separation of concerns, in the past these have been mixed and has lead to confusing situations (e.g. being able to express things in the application configuration accidentally that should have not been allowed).

The mapping is not ideal, and there [are more automated options](https://github.com/anchore/go-struct-converter), however, I wanted to make the understandability as simple as possible (e.g., here's a function where the mapping is done).

I've tried to reduce error prone issues such as forgetting to add something to the mapping (which I could see being a popular issue) with testing. The testing has more complexity (instead of the mapping code) but helps defend against brittleness.